### PR TITLE
Refactor on controllers

### PR DESCRIPTION
**Pull request by hmuir**
Refactor on Budget-RentaCar's controllers

### CODE REVIEW

File: www/app/carView/carView.controller.js [hunk 1]
```diff
@@ -3,11 +3,11 @@
...
-    .controller('CarViewController', CarViewController);
+    .controller('CarViewCtrl', CarViewCtrl);
 
   /* @ngInject */
 
-  function CarViewController($scope,
+  function CarViewCtrl($scope,
                              $state,
                              CarViewService,
                              CarInfoFirebaseService,
```
Comment (pernixgit) [inline www/app/carView/carView.controller.js]: Fix indentation.

File: tests/e2e/carDeliveryInfo/carDeliveryInfo.spec.js [hunk 2]
```diff
@@ -15,6 +15,6 @@ describe('Car Delivery Information page', function() {
 
   it('should go to carView page', function() {
     element(by.css(".button-primary")).click();
-    expect(browser.getCurrentUrl()).toEqual("http://localhost:8100/#/carview")
+    expect(browser.getCurrentUrl()).toEqual("http://localhost:8100/#/ ")
```
Comment (johnnyxu-pernix) [inline tests/e2e/carDeliveryInfo/carDeliveryInfo.spec.js]: Why did you change the path?
